bin/xbps-install/main.c

```diff
@@ -107,6 +108,7 @@ main(int argc, char **argv)
 		{ "help", no_argument, NULL, 'h' },
 		{ "ignore-conf-repos", no_argument, NULL, 'i' },
 		{ "ignore-file-conflicts", no_argument, NULL, 'I' },
+		{ "trust-disk-space", no_argument, NULL, 'T' },
```

Comment:
Trust doesn't feel like the correct word; also, if it's going to be a command line arg I think it should exclusively be a long opt; but Duncaen has expressed preference for an env var.